dev-lang/erlang/erlang-21.0.5.ebuild

```diff
@@ -0,0 +1,220 @@
+
+	local aclocal_master="./erts/aclocal.m4"
+	local install_sh_master="./erts/autoconf/install-sh"
+	local config_guess_master="./erts/autoconf/config.guess"
+	local config_sub_master="./erts/autoconf/config.sub"
+
+	for dir in ${aclocal_dirs}; do
+		"${install_sh_master}" -m 644 -t "${dir}" "${aclocal_master}" || die
+	done
+
+	for dir in ${autoconf_aux_dirs}; do
+		"${install_sh_master}" -d "${dir}" || die
+		"${install_sh_master}" -t "${dir}" "${install_sh_master}" || die
+		"${install_sh_master}" -t "${dir}" "${config_guess_master}" || die
+		"${install_sh_master}" -t "${dir}" "${config_sub_master}" || die
+	done
+}
+
+src_prepare() {
+	default
+
+	# Determines which directories to recurse into with autoconf
+	erlang_create_lib_configure_in
+
+	# Move local autoconf files into the neccessary directories
+	erlang_distribute_config_helpers
+
+	java-pkg-opt-2_src_prepare
+
+	eautoreconf
+}
+
+src_configure() {
+	use wxwidgets && setup-wxwidgets
+
+	econf \
+		--disable-builtin-zlib \
+		$(use_enable hipe) \
+		$(use_enable kpoll kernel-poll) \
+		$(use_with java javac) \
+		$(use_with odbc) \
+		$(use_enable sctp) \
+		$(use_with ssl) \
+		$(use_with ssl ssl-rpath "no") \
+		$(use_enable ssl dynamic-ssl-lib) \
+		$(use_enable systemd) \
+		$(use_enable pgo) \
+		$(usex wxwidgets "" "--with-wxdir=/dev/null") \
+		--enable-threads
+}
+
+src_compile() {
+	emake
+
+	if use emacs ; then
+		pushd lib/tools/emacs &>/dev/null || die
+		elisp-compile *.el
+		popd &>/dev/null || die
+	fi
+}
+
+extract_version() {
+	sed -n -e "/^$2 = \(.*\)$/s::\1:p" "${S}/$1/vsn.mk"
+}
+
+src_install() {
+	local ERL_LIBDIR="/usr/$(get_libdir)/erlang"
+	local ERL_LIBDIR_REL="$(get_libdir)/erlang"
+	local ERL_INTERFACE_VER="$(extract_version lib/erl_interface EI_VSN)"
+	local ERL_ERTS_VER="$(extract_version erts VSN)"
+	local MY_MANPATH="/usr/share/${PN}/man"
+
+	[[ -z "${ERL_ERTS_VER}" ]] && die "Couldn't determine erts version"
+	[[ -z "${ERL_INTERFACE_VER}" ]] && die "Couldn't determine interface version"
+
+	emake INSTALL_PREFIX="${ED}" install
+
+	if use doc ; then
+		local DOCS=( "AUTHORS" "HOWTO"/* "README.md" "CONTRIBUTING.md" "${WORKDIR}"/doc/. "${WORKDIR}"/lib/. "${WORKDIR}"/erts-* )
+		docompress -x /usr/share/doc/${PF}
+	fi
+
+	einstalldocs
+
+	dosym "../${ERL_LIBDIR_REL}/bin/erl" /usr/bin/erl
+	dosym "../${ERL_LIBDIR_REL}/bin/erlc" /usr/bin/erlc
+	dosym "../${ERL_LIBDIR_REL}/bin/escript" /usr/bin/escript
+	dosym "../${ERL_LIBDIR_REL}/lib/erl_interface-${ERL_INTERFACE_VER}/bin/erl_call" /usr/bin/erl_call
+	dosym "../${ERL_LIBDIR_REL}/erts-${ERL_ERTS_VER}/bin/beam.smp" /usr/bin/beam.smp
+
+	## Clean up the no longer needed files
+	rm "${ED}/${ERL_LIBDIR}/Install" || die
+
+	insinto "${MY_MANPATH}"
+
+	doins -r "${WORKDIR}"/man/*
+
+	# extend MANPATH, so the normal man command can find it
+	# see bug 189639
+	echo "MANPATH=\"${MY_MANPATH}\"" > "${T}/90erlang" || die
+	doenvd "${T}/90erlang"
+
+	if use emacs ; then
+		pushd "${S}" &>/dev/null || die
```

Comment:
I'm confused. Why wouldn't you be in `${S}` at this point?

Reply:
fixed